customTests/azure_gpu_bandwidth.nhc

```diff
@@ -173,8 +169,187 @@ function check_gpu_bw()
         result_lines_array["$current_test"]="$extracted_lines"
     fi
 
-    evaluate_result $EXP_CUDA_BW $EXP_P2P_BW
+    evaluate_nvBW_result $EXP_CUDA_BW $EXP_P2P_BW
  
     return 0
 }
 
+function get_OAM_pairs(){
+    # Get OAM pairs
+    # return array where the index is the GPU id and the value is the OAM pair
+    oam_check=$(rocm-smi --showserial | grep GPU)
+
+    if [[ "$oam_check" == *"Not supported"* ]]; then
+        echo ""
+        return 0
+    fi
+
+    mapfile -t gsn < <(rocm-smi --showserial | grep GPU | awk '{print $5}')
+
+    oam_pairs=()
+
+    for ((i=0; i<${#gsn[@]}; i++)); do
+        for ((j=0; j<${#gsn[@]}; j++))
+        do
+            if [[ $i -eq $j ]]; then
+                continue
+            else
+                if [[ "${gsn[$j]}" == *"${gsn[$i]}"* ]]; then
+                    oam_pairs+=($j)
+                fi
+            fi
+        done
+    done
+    echo "${oam_pairs[@]}"
+    # check if the array is empty
+    if [ ${#oam_pairs[@]} -eq 0 ]; then
+        echo ""
+    fi
+    return 0
+}
+
+function check_rocm_gpu_bw(){
+    # Check GPU BW using ROCM Bandwidth Test 
+    exp_hd_bw=$1
+    exp_dd_bw=$2
+    exp_oam_bw=$3
+
+    if [ ! -f /opt/rocm/bin/rocm-bandwidth-test ]; then
+        die 1 "check_gpu_bw: rocm_bandwidth_test not found. Ensure ROCM is installed and sourced correctly."
```

Comment:
might want to specifically call out that you may need to install rocm_bandwidth_test if ROCm is installed correctly. Also ROCM -> ROCm.

Reply:
Thanks! Will address

Reply:
I do install it with AZ NHC. Let me specify to check the install.

Reply:
addressed